### PR TITLE
Feature/cplp 843/show all data

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**

### CODE REVIEW

**Review by jjeroch — COMMENTED** [review]
Looks good. Only one minor request added regarding the application status "CREATED"

File: src/components/dragdrop.tsx [hunk 1]
```diff
@@ -21,13 +24,26 @@ interface DragDropProps {
 export const DragDrop = ({
   currentActiveStep,
   addCurrentStep,
-  addFileNames,
 }: DragDropProps) => {
   const { t } = useTranslation()
+  const dispatch = useDispatch()
+
+  const { status, error } = useSelector(applicationSelector)
+  const obj = status[status.length-1] //.find(o => o['applicationStatus'] === CREATED);
```
[the remark this answers is not in the document]
Comment (nidhigarg-bmw) [reply src/components/dragdrop.tsx:32]: @jjeroch Actually this status get updated on landing page due to which we not to track "obj" according to status "CREATED" on other steps of registration form.

File: src/components/dragdrop.tsx [hunk 2]
```diff
@@ -36,29 +52,8 @@ export const DragDrop = ({
...
-      const values = await Promise.all<Response>(uploadRequests)
-      if (values.every((value) => value.ok)) {
-        toast.success('All files uploaded')
-        addFileNames(files.map((file) => file.file.name))
-      } else {
-        if (values.every((val) => val.status === values[0].status)) {
-          const errorCode = values[0].status
-          const message = DataErrorCodes.includes(errorCode)
-            ? t(`ErrorMessage.${errorCode}`)
-            : t(`ErrorMessage.default`)
```
Comment (oyo) [inline src/components/dragdrop.tsx:51]: you could call that key `ErrorMessage.undefined` then you don't need the  ? :  operator and can just go with line 50

File: src/components/dragdrop.tsx [hunk 3]
```diff
@@ -95,6 +90,21 @@ export const DragDrop = ({
             PreviewComponent={props => <DragdropFiles props={props} />}
           />
         </div>
+        <div className="documents-data mx-auto col-9 mt-4">
+          {
+            documents.map((document: any, index) =>
```
Comment (oyo) [inline src/components/dragdrop.tsx:95]: we should avoid using `any` - is there no type for `document` defined?

File: src/components/verifyRegistration.tsx [hunk 4]
```diff
@@ -24,8 +27,21 @@ export const VerifyRegistration = ({
 }: VerifyRegistrationProps) => {
   const { t } = useTranslation()
 
-  const { companyDetails } = useSelector(applicationSelector)
-  console.log('companyDetails', companyDetails)
+  const dispatch = useDispatch()
+
+  const { status, error, companyDetails } = useSelector(applicationSelector)
+  const { registrationData } = useSelector(stateSelector)
+  console.log('registrationData', registrationData)
```
Comment (oyo) [inline src/components/verifyRegistration.tsx:34]: `console.log` statements like this we can use during development but they should be removed before commit

File: src/components/verifyRegistration.tsx [hunk 5]
```diff
@@ -24,8 +27,21 @@ export const VerifyRegistration = ({
...
+  
+  const obj = status[status.length-1] //.find(o => o['applicationStatus'] === CREATED);
```
Comment (oyo) [inline src/components/verifyRegistration.tsx:36]: that comment should be removed or add a `TODO` comment if you want to activate that later

File: src/components/verifyRegistration.tsx [hunk 6]
```diff
@@ -121,21 +138,21 @@ export const VerifyRegistration = ({
                   <span className="col-6">
                     {t('verifyRegistration.street')}
                   </span>
-                  <span className="col-6">{companyDetails?.streetName}</span>
+                  <span className="col-6">{registrationData?.streetName}{' '}{registrationData?.streetNumber}</span>
```
Comment (oyo) [inline src/components/verifyRegistration.tsx:141]: I think the `}{' '}{` can be written as `} {`

File: src/components/verifyRegistration.tsx [hunk 7]
```diff
@@ -121,21 +138,21 @@ export const VerifyRegistration = ({
...
                 </Row>
               </li>
               <li className="list-group-item-cax">
                 <Row>
                   <span className="col-6">{t('verifyRegistration.city')}</span>
-                  <span className="col-6">{companyDetails?.city}</span>
+                  <span className="col-6">{registrationData?.zipCode}{' '}{registrationData?.city}</span>
```
Comment (oyo) [inline src/components/verifyRegistration.tsx:147]: see comment above

File: src/state/features/applicationDocuments/api.ts [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+      `/api/registration/application/${applicationId}/documentType/${documentTypeId}/documents`,
+      RequestService.getHeaders()
+    )
+
+  public postDocument = (applicationId: string, documentTypeId: string, file: any) =>{
+    const formdata = new FormData()
+    formdata.append('document', file)
+    return fetch(`${getApiBase()}/api/registration/application/${applicationId}/documentType/${documentTypeId}/documents`, {
+      method: 'POST',
+      headers: {
```
Comment (oyo) [inline src/state/features/applicationDocuments/api.ts:31]: use `RequestService.getHeaders()` here just like above in line 23

File: src/state/features/applicationVerifyRegister/api.ts [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    return this.classInstance
+  }
+
+  public getRegistrationData = (applicationId: string) => {
+    return this.instance.get<RegistrationDetails>(
+      `/api/registration/application/${applicationId}/registrationData`,
+      {
+        headers: {
```
Comment (oyo) [inline src/state/features/applicationVerifyRegister/api.ts:25]: use `RequestService.getHeaders()`

File: src/state/features/applicationVerifyRegister/api.ts [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+      }
+    )
+  }
+
+  public submitRegistration = () => {
+    return this.instance.post<string>(
+      `/api/registration/submitRegistration`,
+      {},
+      {
+        headers: {
```
Comment (oyo) [inline src/state/features/applicationVerifyRegister/api.ts:37]: use `RequestService.getHeaders()`

**Review by oyo — CHANGES_REQUESTED** [review]

**Comment by nidhigarg-bmw** [comment]
Hi @oyo 
Please review this PR again. I made the changes.

**Review by oyo — APPROVED** [review]
looks good now

**Pull request merged**